pkg/artifacts/signable.go

```diff
@@ -175,6 +154,78 @@ func ExtractOCIImagesFromResults(tr *v1beta1.TaskRun, logger *zap.SugaredLogger)
 	return objs
 }
 
+// ExtractIntotoSignableTargetFromResults extracts signable targets that aim to generate intoto provenance as materials within TaskRun results and store them as structuredSignable.
+func ExtractIntotoSignableTargetFromResults(tr *v1beta1.TaskRun, logger *zap.SugaredLogger) []interface{} {
```

Comment:
unexport

Reply:
I think I wanted to use this as artifacts. ExtractIntotoSignableTargetFromResults in intotoite6.go